.circleci/config.yml

```diff
@@ -128,12 +136,25 @@ jobs:
             ${HOME}/.npmrc
       - run:
           name: Bump version
-          command: npx athloi version ${CIRCLE_TAG} --filter "dotcom-*"
+          command: npm version ${CIRCLE_TAG} --workspace=packages/
       - run:
           name: NPM publish
-          command: |
-            export DIST_TAG=$(node scripts/get-npm-dist-tag.js $CIRCLE_TAG)
-            if [[ $DIST_TAG ]]; then npx athloi publish -- --access=public --tag=${DIST_TAG}; fi
+          command: npm publish --access=public --tag=latest
```

Comment:
**thought**: the only downside to this approach is that if we do a patch release for an older breaking version then that release will be tagged as `latest`. but I don't think that it's a big deal and we get the benefits of clearer configuration with more logic staying in the CircleCI configuration file, thank you!

Reply:
We are losing that functionality however it would be something we rarely do and our aim with Page Kit is that everyone is using the same version.